### DISCUSSION TITLE
Photo folder

### DISCUSSION THREAD

**Discussion by limafilho** [photoprism/photoprism#1686] [General]
Hi, I just installed Photoprism on my Raspberry Pi 4 and everything is running ok. I get browser access normally.

I uploaded a photo from my computer which is on the same local network and the photo is already appearing in the Photoprism review folder. But I would really like to know where the photo is on the disk. In the file "docker-compose.yml" it is set to be "/home/pi/Pictures", but there is no picture there. And one more question...when you upload, you are actually copying and not moving the photo to Photoprism, correct!?

**Comment by graciousgrey** [comment]
Hi,

The photos you upload using the web upload (https://docs.photoprism.org/user-guide/library/upload/) will be added to PhotoPrism's originals directory in subfolders with the structure year/month.

So in case your docker-compose file looks like this: 
` - "/home/pi/Pictures:/photoprism/originals"`
and the file you uploaded was taken in July 2021 it's location is: /home/pi/Pictures/2021/07 outside of the container or /photoprism/originals/2021/07 within the container.

Yes, when using the web upload this copies the file.

**Comment by limafilho** [comment]
Hi Theresa. Thanks for your response
I did exactly what you told me to do, but I was not successful. I uploaded some pictures from my cell phone, but when I checked the folder, there was nothing there.

Am I doing something wrong?!

![photoprism1](https://user-images.githubusercontent.com/15817054/139598752-bc624f42-e98e-4b68-a42c-311bc8a5d3b6.PNG)
![photoprism2](https://user-images.githubusercontent.com/15817054/139598754-84ae8075-8d03-40b6-9055-63f4529e4c23.PNG)

**Reply by lastzero** [reply]
Maybe the container is running as `root` but you are browsing files as `pi`? In that case, you may not see them. Use the command `sudo chmod -R a+rwx /home/pi/Pictures` to make files readable for everyone - or start the container as the user `pi`. If you do that, make sure existing files and folders have the right permissions as otherwise PhotoPrism won't have access to its own files as well.

See https://docs.photoprism.org/getting-started/troubleshooting/
